equality.go

```diff
@@ -1,7 +1,10 @@
 // Package assert - makes sure that 2 things are the same
 package assert
 
-import "time"
+import (
+	m "math"
```

Comment:
I think using short one letter abbreviations makes code harder to read. Would you mind using full name of the package?

Reply:
sure, it is just used for math.Abs(). 
(Otherwise it is annoying to use the long name in computations, when using several math functions in one equation -> looks awkward for mathematicians  ;-)